i18n/en.json

```diff
@@ -2597,14 +2598,14 @@
   "invitation_modal.invite_members.title": "Invite **Members**",
   "invitation_modal.members.invite_button": "Invite Members",
   "invitation_modal.members.or": "OR",
-  "invitation_modal.members.search_and_add.title": "Invite People",
+  "invitation_modal.members.search_and_add.title": "Add or Invite People",
   "invitation_modal.members.search-and-add.description": "Search and add members from other teams or email invite new users.",
```

Comment:
Propose for clarity: "...email invite new users." > "...send email invitations to new users."

Reply:
I can't find the string, but for the equivalent guests screen the current line states "Search and add guests or email invite new users." which isn't super clear to me what "search and add guests" means. In my mind it should convey:

1. Guests are being added to this team from users that may exist in other teams
2. New email invites can be send to add guests to this team 

0/5, perhaps something like, or break it into two sentences:

"Search and add guests from other teams or send email invitations to new guests."

Reply:
Side note, 0/5 if we standardize on the work "invites" vs "invitations"

Reply:
Asking Justine to weigh in on this (not able to directly add her, will message outside of this PR) Here are some thoughts from Mike, Dennis & I: 

For members:  Add existing members or send email invites to new members.

For Guests:  Add existing guests or send email invites to new guests.

Reply:
I like that suggestion, @wiersgallak. I can't think think of any other phrasing that could be clearer. I think "invites" is a better option as we use it in the existing documentation and UI already so it's a more natural fit.

Reply:
0/5 but "Add existing members" and "Add existing guests" doesn't convey in my mind that are existing users from other teams, which may lead to confusion about the purpose of the action they are executing. But I don't want to block this PR on a text change so just leaving as food for thought.